src/components/forms/AddressInput/index.tsx

```diff
@@ -53,11 +53,9 @@ const AddressInput = ({
     [validators],
   )
 
-  const allValidators = useCallback(
-    (val: string) => {
-      // Internal validators + externally passed validators
-      return composeValidators(required, mustBeEthereumAddress, sanitizedValidators)(val)
-    },
+  // Internal validators + externally passed validators
+  const allValidators = useMemo(
```

Comment:
Also improved this, useMemo is better in this case because I don't need to pass `val` explicitly.